js/background.js

```diff
@@ -277,4 +280,38 @@
             timestamp: timestamp, source: pushMessage.source
         });
     }
+
+    var App = Backbone.Model.extend({
+        initialize: function(opts){
+            this.inboxView = null;
+        },
+        getAppView: function(destWindow){
+
+            var self = this;
+
+            return ConversationController.updateInbox().then(function() {
+                try {
+                    if (self.inboxView) { self.inboxView.remove(); }
+                    self.inboxView = new Whisper.InboxView({model: self, window: destWindow});
+                    window.openConversation = function(conversation) {
```

Comment:
`s/window/destWindow`

References to `window` previously executed in the context of index.js/index.html, must be changed to `destWindow` here. In particular, this breaks opening a conversation from a notification click.